assessmentModel/src/commonMain/kotlin/serialization/Results.kt

```diff
@@ -37,7 +37,9 @@ data class AnswerResultObject(override val identifier: String,
                               override var startDateTime: Instant = Clock.System.now(),
                               @SerialName("endDate")
                               @Serializable(with = InstantSerializer::class)
-                              override var endDateTime: Instant? = null)
+                              override var endDateTime: Instant? = null,
+                              override val questionText: String? = null,
+                              val questionData: JsonElement? = null)
```

Comment:
What is questionData used for?

Reply:
At one point there was a request to include additional data in the result set about the questions other than just the question text. The compromise was to add "questionData" as `Any` object.